libevmasm/KnownState.cpp

```diff
@@ -91,6 +91,12 @@ KnownState::StoreOperation KnownState::feedItem(AssemblyItem const& _item, bool
 	{
 		// can be ignored
 	}
+	else if (_item.type() == AssignImmutable)
```

Comment:
Why no special case about reading from immutable?

Reply:
Reading from immutables during creation uses plain reads from memory and no special assembly items - reading during runtime with ``PushImmutable`` basically behaves like ``PushLibraryAddress``, so the ``else if (_item.type() != Operation)`` case should be fine for that.